pkgs/tools/misc/grub/2.0x.nix

```diff
@@ -67,7 +67,7 @@ stdenv.mkDerivation rec {
 
     # Pull upstream patch to fix linkage against binutils-2.36.
     (fetchpatch {
-      name = "binutils-2.36";
+      name = "binutils-2.36.patch";
```

Comment:
:heavy_check_mark: (see the commit for details if you're reviewing and confused)